src/Tracy/Template/MailPanel.phtml

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Contributte\Mail\Tracy\Template;
+
+use Tracy\Dumper;
+
+?>
+
+<h1>Contributte/Mail</h1>
```

Comment:
Could we rename it to Mails?

Reply:
:rofl: sure but extension's name is mail

Reply:
@f3l1x done